src/protocol/response_writer.rs

```diff
@@ -1,3 +1,6 @@
+use alloc::vec::Vec;
+use alloc::string::String;
```

Comment:
These imports will need to be `cfg` gated, as this will result in a compilation error in non `alloc` builds.

Reply:
Fixed